src/pages/en/migrate.md

```diff
@@ -9,6 +9,31 @@ This guide will help you migrate from older versions of Astro to the latest vers
 
 Read the guide below for major highlights and instructions on how to handle breaking changes.
 
+## Upcoming Astro 1.0 Release Candidate
+
+We plan to release an Astro v1.0 Release Candidate (RC) sometime in the next several days. This version is not yet available to most users, however you can try it yourself today by running this command:
+
+```
+npm install astro@next--rc
+```
+
+Below are the changes that you should be aware of when migrating to the Astro v1.0 RC.
```

Comment:
This is fine but if it’s going to change imminently, maybe we future-proof it from the off? This structure would let us simply delete the note once that’s no longer needed.

````suggestion
## Astro 1.0 Release Candidate

The Astro v1.0 Release Candidate (RC) introduces some changes that you should be aware of when migrating from beta or earlier releases. See below for more details.

:::note
The Release Candidate is not yet installed by default for most users, but you can try it today by running this command:

```sh
npm install astro@next--rc
```
:::
````